app/components/windows/AddSource.vue.ts

```diff
@@ -65,11 +67,16 @@ export default class AddSource extends Vue {
     if (this.isTaken(this.name)) {
       this.error = 'That name is already taken';
     } else {
-      const sceneItem = this.scenesService.activeScene.createAndAddSource(
+      const source = this.sourcesService.createSource(
         this.name,
-        this.sourceType
+        this.sourceType,
+        {},
+        {
+          propertiesManager: this.propertiesManager ? this.propertiesManager : void 0
```

Comment:
Do we really need explicitly set which property manager we have to use? What will happen if I try to use `Default` property manager for `widget` or `streamlabels` ?

Reply:
Yes.  The properties manager determines whether it is streamlabels or not.  For example:
- A "normal" text source: `text_gdiplus` source with `default` manager
- A "Stream Label": `text_gdiplus` source with `streamlabel` manager
- A "normal" browser source: `browser_source` with `default` manager
- A "Widget": `browser_source` with `widget` manager